pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -365,18 +366,29 @@ type InfoList struct {
 type Project struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+	Spec              ProjectSpec   `json:"spec,omitempty"`
 	Status            ProjectStatus `json:"status,omitempty"`
 }
 
+type ProjectSpec struct {
+	DefaultRegion    string   `json:"defaultRegion,omitempty"`
+	SupportedRegions []string `json:"supportedRegions,omitempty"`
+}
+
 type ProjectStatus struct {
-	Placement string `json:"placement,omitempty"`
-	Namespace string `json:"namespace,omitempty"`
+	Placement     string `json:"placement,omitempty"`
+	Namespace     string `json:"namespace,omitempty"`
+	DefaultRegion string `json:"defaultRegion,omitempty"`
 }
 
 func (in *Project) NamespaceScoped() bool {
 	return false
 }
 
+func (in *Project) ForRegion(region string) bool {
```

Comment:
I am fine changing the names of these methods, but they have to have the same name because an interface is used in Hub to detect when objects are "regioned"

If it's OK with you, we can merge this and adjust the name in the next set of changes.

Reply:
Sounds good to me!